File: tutorials/testing/java/tut_worldmap_p1.java
import java.util.ArrayList;
import java.util.Map;
import bridges.connect.Bridges;
import bridges.connect.DataSource;
import bridges.data_src_dependent.Country;
import bridges.base.WorldMap;


// This tutorial illustrates how to access the data of the World map with 
// boundaries; the countries are filled with default color
public class tut_worldmap_p1 {
	public static void main(String[] args) throws Exception {

		// create Bridges object
		Bridges bridges = new Bridges (YOUR_ASSSIGNMENT_NUMBER, "YOUR_USER_ID", 
												"YOUR_API_KEY");
		// set title, description
		bridges.setTitle("Tutorial: Drawing US Map with State Boundaries");
		bridges.setDescription("All US states, No Counties");

		// get data source object
		DataSource ds = bridges.getDataSource();

		// get us map data - states and boundaries

		WorldMap wm = new WorldMap();

		// set data structure
		bridges.setMap(wm);
		bridges.setDataStructure(wm);

		// visualize
		bridges.visualize();
	}
}

File: tutorials/testing/python/tut_worldmap_p1.py
from bridges.bridges import *
from bridges.data_src_dependent import *
from bridges.dl_element import *
from bridges.world_map import *

def main():
    # create the Bridges object, set credentials
    bridges = Bridges(YOUR_ASSSIGNMENT_NUMBER, "YOUR_USER_ID", "YOUR_API_KEY")

    # set title, description
    bridges.set_title("World Map Using BRIDGES")
    bridges.set_description("Draws all countries and their boundaries in a default color")

    # with no arguments to the function, gets all countries of the world
    country_data = get_world_map_data()
    wm = WorldMap()

    #visualizes world map
    bridges.set_map(wm);
    bridges.set_data_structure(wm)
    bridges.visualize()

if __name__ == '__main__':
    main()
